docs/useful_tools.md

```diff
@@ -258,7 +258,19 @@ You should obtain a respose similar to:
   }
 ]
 ```
-
+And you can use `test_torchserver.py` to compare result of torchserver and pytorch, and visualize them.
+```shell
```

Comment:
Need to leave an empty line before and after code block following markdown style guidance.